.github/workflows/build_test_invoke.yml

```diff
@@ -101,15 +101,15 @@ jobs:
             type: 'Invoke'
             file: 'tests/integration/build_invoke/python/test_python_3_11.py'
     env:
-      PYTHON_VERSION_INSTALL: ${{ matrix.version }}
+      PYTHON_VERSION_INSTALL: 3.8
```

Comment:
I wonder if this is redundant since we have the same env var defined at the root scope.

Reply:
Thats a good point, will remove this since this was used before to install SAM CLI with the same version that was testing the runtime